src/urbit/app/chess.hoon

```diff
@@ -526,11 +526,11 @@
         ==
       =/  fen  (position-to-fen u.new-position)
       :-  :~  :*  %give  %fact  ~[/game/(scot %da game-id.action)/updates]
-                  %chess-update  !>([%position game-id.action fen |])
+                  %chess-update  !>([%position game-id.action fen (gte ply-50-move-rule.u.new-position 100)])
```

Comment:
I just realized we're replicating the `check-50-move-rule` logic - we should be calling `check-50-move-rule` instead:
```
:-  :~  :*  %give  %fact  ~[/game/(scot %da game-id.action)/updates]
            %chess-update  !>([%position game-id.action fen (check-50-move-rule u.new-position)])
```